cli/ballerina-packerina/src/main/java/org/ballerinalang/packerina/init/models/SrcFile.java

```diff
@@ -56,25 +65,34 @@
                                                   "}";
 
     private static final String MAIN_FUNCTION_CONTENT = "import ballerina/io;\n" +
+                                                        "documentation {\n" +
```

Comment:
Should there be an empty line between the import and the documentation

Reply:
Added an empty line with 0af28ab